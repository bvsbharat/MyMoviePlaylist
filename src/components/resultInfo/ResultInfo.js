import React from "react";
import PropTypes from "prop-types";

const ResultInfo = ({ totalResults, showing }) => {
    return (
        <div className="my-10 border-t border-gray-200">
            {totalResults && (
                <div className="flex items-center my-2 mx-2">
                    <p className="text-gray-500">
                        {`  Showing 1 - ${showing} of ${totalResults} results`}{" "}
                    </p>
                </div>
            )}
        </div>
    );
};

ResultInfo.defaultProps = {
    totalResults: 0,
    showing: 0,
};

ResultInfo.propTypes = {
    totalResults: PropTypes.number,
    showing: PropTypes.number,
};

export default ResultInfo;
