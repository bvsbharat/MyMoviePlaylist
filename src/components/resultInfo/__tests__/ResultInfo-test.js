import mount from "@test";
import renderer from "react-test-renderer";
import ResultInfo from "../ResultInfo";

describe(ResultInfo, () => {
    const component = mount(<ResultInfo />);

    it("renders without crashing", () => {
        expect(component).toBeTruthy();
        const componentRender = renderer.create(<ResultInfo />);
        const tree = componentRender.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
