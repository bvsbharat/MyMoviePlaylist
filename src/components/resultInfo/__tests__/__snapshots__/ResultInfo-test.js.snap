// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultInfo renders without crashing 1`] = `
<div
  className="my-10 border-t border-gray-200"
>
  0
</div>
`;
